### ISSUE TITLE
Various keybindings no longer work

### ISSUE THREAD

**Issue by HaydenReeve** [11cafe/comfyui-workspace-manager#319]
This is a really odd one.

Having this setting set in your options means you can no longer type "S":

![image](https://github.com/11cafe/comfyui-workspace-manager/assets/39004735/98c13cc4-d294-4202-923f-53afb46eec9d)

Not in search, not in prompt nodes, not anywhere...

I also seem to have lost the ability to press "enter" on a prompt node. Only Shift+Enter appears to work now. Nor can I use the "down arrow key" anymore to navigate long prompts.

Might've caught a mid patch here when I pulled the most recent changes this afternoon. Oops.

**Comment by Weixuanf** [comment]
https://github.com/11cafe/comfyui-workspace-manager/pull/321/files

should be fixed now, sorry about it 🤦‍♀️

**Comment by HaydenReeve** [comment]
Don't be sorry, you are doing fantastic work. I've just pulled the latest and all issues with the `enter` and `arrow keys` are all fixed. 

There is still the hilarious issue where your save workflow hotkey will override any text box input, so you can't use `S` if you have bound save workflow to `Shift + S` but that's much less inconvenient. I can always change the hotkey.

Thank you again! :)

**Comment by Weixuanf** [comment]
> Don't be sorry, you are doing fantastic work. I've just pulled the latest and all issues with the `enter` and `arrow keys` are all fixed.
> 
> There is still the hilarious issue where your save workflow hotkey will override any text box input, so you can't type the character `S` if you have bound save workflow to `Shift + s` but that's much less inconvenient. I can always change the hotkey.
> 
> Thank you again! :)

Yeah because the control+S key binding is occupied by ComfyUI's default save workflow. So we need to think of something else 🤔 maybe alt+S is better?

**Comment by HaydenReeve** [comment]
It appears that no matter what you change that keybinding to, if it is a manifestation of `shift + letter` that keybind takes precedence over any text input comfy might be focused on.

So, changing it to `shift + D` means you can no longer type `D` etc.
